merlin-sdk/src/main/java/gov/nasa/jpl/aerie/merlin/protocol/model/TaskSpecType.java

```diff
@@ -18,6 +20,8 @@ Specification instantiate(Map<String, SerializedValue> arguments)
   List<String> getValidationFailures(Specification taskSpec);
 
   Task createTask(Model model, Specification taskSpec);
+  ValueSchema getReturnValueSchema();
+  SerializedValue serializeReturnValue(Object returnValue);
```

Comment:
Thinking out loud: my initial reaction is to suggest changing this to
```java
<T> SerializedValue serializeReturnValue(T returnValue);
```
since I generally avoid using a bare `Object` parameter type. But I can't think of a good reason why `Object` isn't fine here.

Any chance you could run through the changes made in this PR sometime next week? Would be cool to go over the generated code that's implementing these methods.